moviesurfer/data/download_data.py

```diff
@@ -0,0 +1,36 @@
+import logging
+import os
+import zipfile
+
+import wget
+
```

Comment:
logger = logging.getLogger(__name__)

Reply:
See in the recent commit. Tried to handle this issue, although might have made mistakes here and there:)